packages/opc-header/dist/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>opc-header</title>
+    <script src="opc-header.js"></script>
+    <style>
+        body {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        .test__styles {
+            font-size: 1rem;
+        }
+    </style>
+</head>
+<body>
+    <opc-header style="--opc-header--width: 1200px;">
+        <div slot="breadcrumb-slot">
+        </div>
+        <div slot="right-link-slot" class="test__styles">
+        </div>
+    </opc-header>
+
+    <opc-header style="--opc-header--width: 1200px;">
+        <div slot="breadcrumb-slot">
+            This is a breadcrumb
+        </div>
+        <div slot="right-link-slot" class="test__styles">
+        </div>
+    </opc-header>
+
+    <opc-header style="--opc-header--width: 1200px;">
+        <div slot="breadcrumb-slot">
+            This is a breadcrumb
+        </div>
+        <div slot="right-link-slot" class="test__styles">
+            Video Guide  FAQs  Documentation
```

Comment:
Is the template supposed to be inserted here? If so, how exactly would this work?